### PR TITLE
Set sign-out link to value of settings.LOGOUT_URL.

### PR DESCRIPTION
**Pull request by abztrakt**

### CODE REVIEW

File: foodalert/static/foodalert/js/components/generic-page.vue [hunk 1]
```diff
@@ -63,6 +64,7 @@ export default {
   data() {
     return {
       notificationState: false,
+      logoutURL: logoutUrl,
```
Comment (erinRochfort99) [inline foodalert/static/foodalert/js/components/generic-page.vue]: I think you may need to use 'this.logoutUrl" here

File: foodalert/templates/base.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <script>
             var netid = '{{ netid }}';
+            var logoutUrl = "{{ logout_url }}"
```
Comment (erinRochfort99) [inline foodalert/templates/base.html]: Should we use double or single quotes for these variables? (just for the sake of consistency)

File: foodalert/static/foodalert/js/components/generic-page.vue (hunk 1 above)
Comment (erinRochfort99) [reply foodalert/static/foodalert/js/components/generic-page.vue]: Or, since logoutUrl is a global variable, I don't think this assignment is necessary

Comment (SarthakSingh31) [reply foodalert/static/foodalert/js/components/generic-page.vue]: logout URL also needs to be added to line 17

**Pull request merged**